roles/ohpc_add_easybuild/tasks/main.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Install pip for system python and build tools
+  yum:
+    state: present
+    name: 
+      - '@Development Tools'
+      - python-pip
+   
+- name: Create user build
+  user:
+    name: build
+
+- name: Change owner and permission of easybuild installation directory
+  file:
+    path: "{{ easybuild_prefix }}"
+    owner: root
+    group: build
+    mode: "g+w"
+    recurse: yes
+
+- name: Download EasyBuild bootstrap script
+  get_url:
+    url: https://raw.githubusercontent.com/easybuilders/easybuild-framework/develop/easybuild/scripts/bootstrap_eb.py
+    dest: /tmp/bootstrap_eb.py
+
+- name: Boostrap EasyBuild
+  shell: |
+      source /opt/ohpc/admin/lmod/lmod/init/bash
+      python bootstrap_eb.py "{{ easybuild_prefix }}"
+  become_user: build
+  args:
+    executable: /bin/bash
+    chdir: /tmp
+
+- name: Add EasyBuild installation path into default MODULEPATH
+  replace:
+    path: /etc/profile.d/lmod.sh
+    regexp: '(MODULEPATH=.*)'
+    replace: '\1:{{ easybuild_prefix }}/modules/all'
```

Comment:
I feel like we need to come up with a better regex here in order to address the [replace module's](https://docs.ansible.com/ansible/2.6/modules/replace_module.html) clarification that "It is up to the user to maintain idempotence by ensuring that the same pattern would never match any replacements made."

I can't think of how to structure what is essentially a negative regex, if the line does not contain this path add it.  Suggestions welcome.